hadoop-hdds/container-service/src/main/java/org/apache/hadoop/ozone/container/keyvalue/KeyValueHandler.java

```diff
@@ -664,6 +677,18 @@ ContainerCommandResponseProto handleDeleteChunk(
     return getSuccessResponse(request);
   }
 
+  private void validateChunkData(ChunkBuffer data, ChunkInfo info)
```

Comment:
There can be many kinds of validations. This makes the code more readable in terms of what will be done.
```suggestion
  private void validateChunkChecksumData(ChunkBuffer data, ChunkInfo info)
```